test/test_suite.py

```diff
@@ -509,3 +510,85 @@ def _literal_round_trip(self, type_, input_, output, filter_=None):
                 if filter_ is not None:
                     value = filter_(value)
                 assert value in output
+
+
+class StringTest(_StringTest):
+    @provide_metadata
+    def _literal_round_trip(self, type_, input_, output, filter_=None):
+        """
+        TODO: Remove this override method once DDL issue resolved in spanner_dbapi.
+
+        SPANNER OVERRIDE:
+
+        Spanner DBAPI does not execute DDL statements unless followed by a
+        non DDL statement, which is preventing correct table clean up.
+        The table already exists after related tests finish, so it doesn't
+        create a new table and when running tests for other data types
+        insertions will fail with `400 Duplicate name in schema: t`.
+        Overriding the tests to create and drop a new table to prevent
+        database existence errors.
+        """
+
+        # for literal, we test the literal render in an INSERT
+        # into a typed column.  we can then SELECT it back as its
+        # official type; ideally we'd be able to use CAST here
+        # but MySQL in particular can't CAST fully
+
+        t = Table("t_string", self.metadata, Column("x", type_))
+        t.create()
+
+        with db.connect() as conn:
+            for value in input_:
+                ins = (
+                    t.insert()
+                    .values(x=literal(value))
+                    .compile(
+                        dialect=db.dialect, compile_kwargs=dict(literal_binds=True),
+                    )
+                )
+                conn.execute(ins)
+                conn.execute("SELECT 1")
```

Comment:
Insert statements are DML, not DDL so this select isn't needed.
```suggestion
```

I noticed this was also in the integer compliance PR after I merged it (whoops!) so it should probably be removed from there too.